src/test/java/htsjdk/samtools/cram/RANSInteropTest.java

```diff
@@ -0,0 +1,321 @@
+package htsjdk.samtools.cram;
+
+import htsjdk.HtsjdkTest;
+import htsjdk.samtools.cram.compression.rans.RANSDecode;
+import htsjdk.samtools.cram.compression.rans.RANSEncode;
+import htsjdk.samtools.cram.compression.rans.RANSParams;
+import htsjdk.samtools.cram.compression.rans.rans4x8.RANS4x8Decode;
+import htsjdk.samtools.cram.compression.rans.rans4x8.RANS4x8Encode;
+import htsjdk.samtools.cram.compression.rans.rans4x8.RANS4x8Params;
+import htsjdk.samtools.cram.compression.rans.ransnx16.RANSNx16Decode;
+import htsjdk.samtools.cram.compression.rans.ransnx16.RANSNx16Encode;
+import htsjdk.samtools.cram.compression.rans.ransnx16.RANSNx16Params;
+import org.apache.commons.compress.utils.IOUtils;
+import org.testng.Assert;
+import org.testng.SkipException;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.ByteBuffer;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+/**
+ * Interop test data is kept in a separate repository, currently at https://github.com/samtools/htscodecs
+ * so it can be shared across htslib/samtools/htsjdk.
+ */
+public class RANSInteropTest extends HtsjdkTest {
+    @Test
+    public void testGetHTSCodecsCorpus() throws SkipException{
```

Comment:
`SkipException` is a `RuntimeException` (not checked), so it doesn't need to be declared (here and in the other methods where its declared).

Reply:
done